aws/import_aws_iam_role_policy_attachment.go

```diff
@@ -0,0 +1,38 @@
+package aws
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/iam"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsIamRolePolicyAttachmentImport(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+	conn := meta.(*AWSClient).iamconn
+	roleName := d.Id()
+
+	resp, err := conn.ListAttachedRolePolicies(&iam.ListAttachedRolePoliciesInput{
+		RoleName: aws.String(roleName),
+	})
+	if err != nil {
+		return nil, err
+	}
+
+	results := []*schema.ResourceData{}
+
+	for _, policy := range resp.AttachedPolicies {
```

Comment:
We strongly prefer to have resources only import resources 1:1 currently -- otherwise this can cause dangerous behavior if the operator is not completely vigilant after import. See also: #5631

Reply:
I should also note this behavior of multiple imports can be very problematic if the resources are implemented across separate modules. It will force the operator to `terraform state rm` the other attachments repeatedly for each module (and likely unexpectedly).